react/react1/week1/ToDoList/src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useState } from "react";
+import toDoList from "./assets/toDoList.json";
+import ToDoList from "./components/ToDoList/ToDoList";
+import "./App.css";
+
+function App() {
+  const [toDos, setToDos] = useState(toDoList);
```

Comment:
Good to see you use the useState hook. The todo list might make more sense as a prop, but nevertheless it's good to se that you've understood how to use states